README.md

```diff
@@ -13,13 +13,21 @@ This module transforms between the JSON syntax accepted by the OpenFGA API and t
 
 ## Table of Contents
 
-- [About OpenFGA](#about-openfga)
-- [Resources](#resources)
-- [Installation](#installation)
-- [Contributing](#contributing)
-- [Community Parsers](#community-parsers)
-- [Author](#author)
-- [License](#license)
+- [OpenFGA Syntax Transformer](#openfga-syntax-transformer)
```

Comment:
```
- [OpenFGA Syntax Transformer](#openfga-syntax-transformer)
```

No need for this, can we remove it and have 1 less indentation for everything else?

Reply:
It is automatically brought in my vscode prettier...let me undo that